lib/internal/util/inspect.js

```diff
@@ -2176,6 +2177,483 @@ function formatBigIntNoColor(bigint, options) {
   );
 }
 
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// https://github.com/denoland/deno/blob/ece2a3de5b19588160634452638aa656218853c5/ext/console/01_console.js#L2575
+const colorKeywords = new SafeMap([
```

Comment:
Non-blocking nit: This adds quite a bit to this one file. I wonder if there's value in moving this out to a separate `lib/internal/util/inspect-colors.js` file or something similar?